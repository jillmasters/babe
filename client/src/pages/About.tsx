import React from 'react';
import { MainView, AboutPage } from '../theme';

import Lottie from 'react-lottie';
import mates from '../animations/mates.json';

const pals = {
  loop: true,
  autoplay: true,
  animationData: mates,
  rendererSettings: {
    preserveAspectRatio: 'xMidYMid slice',
  },
};

const About = () => {
  return (
    <MainView>
      <h4>
        <span role="img" aria-label="confused person emoji">
          ðŸ¤·
        </span>
        Why use Babe?
        <span role="img" aria-label="confused person emoji">
          ðŸ¤·
        </span>
      </h4>
      <AboutPage>
        <h2>
          <span id="babify">Babe </span>is a low-maintenance bill-splitter for
          two people.
        </h2>
        <br />
        <Lottie options={pals} height={200} width={200} />
        <br />
        <h2> It&apos;s for folks with shared lives and sole bank accounts.</h2>
        <br />

        <h5>
          Users get a simple balance, saying which person is owed money overall.
        </h5>
        <h5>
          You can add new, customised bills, view your transaction history, call
          it even, or settle up.
        </h5>
        <h5>
          There&apos;s no fuss, no finicky details, and no direct requests for
          repayment. So you can share your life without invoicing the people you
          love.
        </h5>
      </AboutPage>
    </MainView>
  );
};

export default About;
