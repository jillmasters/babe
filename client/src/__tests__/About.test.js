import React from 'react';
import About from '../pages/About';

import renderer from 'react-test-renderer';

it('About renders correctly', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
