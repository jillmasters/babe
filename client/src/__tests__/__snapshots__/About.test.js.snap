// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<section
  className="css-1rze9zt"
  data-testid="about"
>
  <h4>
    <span
      aria-label="confused person emoji"
      role="img"
    >
      ðŸ¤·
    </span>
    Why use Babe?
    <span
      aria-label="confused person emoji"
      role="img"
    >
      ðŸ¤·
    </span>
  </h4>
  <div
    className="css-h26aco"
  >
    <h2>
      <span
        id="babify"
      >
        Babe 
      </span>
      is a low-maintenance bill-splitter for two people.
    </h2>
    <br />
    <div
      aria-label="animation"
      onClick={[Function]}
      role="button"
      style={
        Object {
          "height": "200px",
          "margin": "0 auto",
          "outline": "none",
          "overflow": "hidden",
          "width": "200px",
        }
      }
      tabIndex="0"
      title=""
    />
    <br />
    <h2>
       It's for folks with shared lives and sole bank accounts.
    </h2>
    <br />
    <h5>
      Users get a simple balance, saying which person is owed money overall.
    </h5>
    <h5>
      You can add new, customised bills, view your transaction history, call it even, or settle up.
    </h5>
    <h5>
      There's no fuss, no finicky details, and no direct requests for repayment. So you can share your life without invoicing the people you love.
    </h5>
  </div>
</section>
`;
